README.md

```diff
@@ -1,19 +1,40 @@
 # 3Box
-User-Managed Application Data for Web3.
+3Box is a distributed database that supports public and private data for Ethereum users. All data is publicly available, but private data can only be decrypted by those to whom the user has provided permission. 
```

Comment:
> private data can only be decrypted by those to whom the user has provided permission.

Change to "private data can only be decrypted by the user".

Reply:
Or maybe you mean something like "private data can only be decrypted by dapps that the user has given explicit permission"?

Reply:
yes @christianlundkvist  the second one. Others (dapps or otherwise) can decrypt my data if I have given them consent